sdk/src/main/java/co/omise/android/models/APIError.kt

```diff
@@ -0,0 +1,18 @@
+package co.omise.android.models
+
+import org.joda.time.DateTime
+
+/**
+ * Represents Error object.
+ *
+ * @see <a href="https://www.omise.co/errors-api">Errors API</a>
+ */
+class APIError(
```

Comment:
Is it possible to use `data class` here?

Reply:
Can't remember why I didn't do this, will add it